### DISCUSSION TITLE
FB&D - Rotten Dreams v4.8 - Bugfix

### DISCUSSION THREAD

**Discussion by Hirxs** [Hirxs-MC/FD-Bedrock-development#187] [Announcements]
![Image](https://hirxs-website.net/wp-content/uploads/2024/10/Screenshot-2024-10-05-181927.png)

# Oh my god! you guys posting an update along with a video!? is this even possible??

## CHANGES
- Modified the hitbox of the "shelf" block so that you can pass under the block

## FIXES
- Fixed a bug where the party hat hitbox was slightly incorrect
- Fixed a bug where when playing multiplayer and sitting in a chair next to a player, they would automatically be teleported to the chair and you would not be able to sit in the chair
- Fixed a bug with the balloon collision hitbox
- Fixed a bug where the "plates" block had an incorrect hitbox
- Fixed a bug where fnaf2 walls didn't have variants (middle and top)
- Fixed a bug where bathroom signs were the same
- Fixed a bug where the red guitar item was not 3D on hand

### FIXES
- Fixed an issue where walls would not have a light dampening (Advanced walls)

<hr /><em>This discussion was created from the release <a href='https://github.com/Hirxs-Workshop/FD-Bedrock-development/releases/tag/official-release-v4.8'>FB&D - Rotten Dreams v4.8 - Bugfix</a>.</em>

**Comment by hannahdaisy1990-coder** [comment]
This is amazing mod i just don't know how to get the new version
